src/wallet/coinselection.cpp

```diff
@@ -529,6 +529,9 @@ class MinOutputGroupComparator
 public:
     int operator() (const OutputGroup& group1, const OutputGroup& group2) const
     {
```

Comment:
Since the goal here is to use the same sort as `descending_effval_weight`, L532 - L 535 can be replaced with this single line:

`return descending_effval_weight(group1, group2);`